cogs/odds_on_cog.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+An example cog to show how things should be done.
+
+Also provides a simple base for starting a new cog.
+"""
+# In this case, discord import is not needed, in some cases it may be.
+import asyncio
+
+import discord
+from discord.ext import commands
+import utils as ut
+
+
+class OddsOnCog(commands.Cog):
+    """Create a class that extends Cog to make our functionality in."""
+
+    def __init__(self, bot):
+        """Save our bot argument that is passed in to the class."""
+        self.bot = bot
+
+    @commands.command(
+        name="oddson",
+        help="Let's you play Odds On against other members")
+    @commands.has_role("Member")
+    async def ping(self, ctx, role, *, wager: str):
```

Comment:
Probably best to rename this coroutine to something more descriptive - I know it's spilled over from the example cog, but still